test/fixtures/linter/autofix/ansi/027_l003_l018_wrong_indent_cte/after.sql

```diff
@@ -0,0 +1,3 @@
+WITH cte AS (
+    SELECT 1
+) SELECT * FROM cte
```

Comment:
Should the rule be changed to be on a new line of it's own?

To fix this to:

```sql
WITH cte AS (
    SELECT 1
)

SELECT * FROM cte
```

Reply:
Interesting. Is there a rule which does that? It seems out of scope for this PR. 🤔

Reply:
Good point. L022 handles that. Maybe should be added to this test config?

Reply:
Sounds good to me. I see these tests as narrowly defined real-world tests, i.e. choose the minimum set of rules to do a nice job on the query.